docs/resampling.ipynb

```diff
@@ -0,0 +1,380 @@
+    "\n",
+    "hst_hdu = fits.open('../data/test_resampling/Cut_HST.fits')\n",
+    "data_hst = hst_hdu[0].data\n",
+    "wcs_hst = WCS(hst_hdu[0].header)\n",
+    "print(wcs_hst.wcs.crval)\n",
+    "wcs_hst.wcs.crval += 2.4750118475607095e-05*np.array([-np.cos(0.4136047623181346 ), -np.sin(0.4136047623181346 )])\n",
+    "print(wcs_hst.wcs.crval)\n",
+    "psf_hst = fits.open('../data/test_resampling/PSF_HST.fits')[0].data\n",
+    "np1,np2 = np.shape(psf_hst)\n",
+    "psf_hst = psf_hst.reshape(1,np1,np2)\n",
+    "\n",
+    "noise = np.concatenate((np.concatenate((data_hst[0,:], data_hst[:,0])),\n",
+    "                                            np.concatenate((data_hst[-1,:], data_hst[:,-1]))))\n",
+    "\n",
+    "n1,n2 = np.shape(data_hst)\n",
+    "data_hst = data_hst.reshape(1, n1, n2).byteswap().newbyteorder()*np.max(data_hsc)/np.max(data_hst)\n",
+    "\n",
+    "r, N1, N2 = data_hsc.shape\n",
+    "\n",
+    "catalog_hst, bg_rms_hst = makeCatalog(data_hst)\n",
+    "catalog_hsc, bg_rms_hsc = makeCatalog(data_hsc)\n",
+    "\n",
+    "#plotting setup\n",
+    "#norm = #scarlet.display.Asinh(img=data_hsc, Q=20)\n",
+    "# Map i,r,g -> RGB\n",
+    "filter_indices = [2,0,1]\n",
+    "\n",
+    "xo,yo = catalog_hst['x'], catalog_hst['y']\n",
+    "\n",
+    "ra, dec = wcs_hst.wcs_pix2world(yo,xo,0)\n",
+    "Yo,Xo, l = wcs_hsc.wcs_world2pix(ra, dec, 0, 0)\n",
+    "\n",
+    "img_rgb = scarlet.display.img_to_rgb(data_hsc)\n",
+    "plt.subplot(121)\n",
+    "plt.imshow(img_rgb)\n",
+    "plt.plot(Xo,Yo, 'o')\n",
+    "plt.subplot(122)\n",
+    "plt.imshow(np.arcsinh(data_hst[0]))\n",
+    "#plt.plot(xo,yo, 'o')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initialize the sources\n",
+    "Each source is a list of fundamental `scarlet.Component` instances and must be based on `scarlet.Source` or a derived class, in this case `ExtendedSource`, which enforces that the source is monotonic and symmetric."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obs_hst = scarlet.Observation(data_hst, wcs = wcs_hst, psfs = None, structure = np.array([0,0,0,1]))\n",
+    "obs_hsc = scarlet.LowResObservation(data_hsc,  wcs = wcs_hsc, psfs = psf_hsc, structure = np.array([1,1,1,0]))\n",
+    "\n",
+    "scene = scarlet.Scene((r+1, n1,n2), wcs = wcs_hst, psfs = psf_hst)\n",
+    "\n",
+    "obs = [obs_hsc, obs_hst]\n",
+    "\n",
+    "bg_rms = np.concatenate((bg_rms_hsc, bg_rms_hst))\n",
+    "\n",
+    "# In[5]:\n",
+    "\n",
+    "sources = [scarlet.CombinedExtendedSource((ra[i], dec[i]), scene, obs, bg_rms, symmetric = 0, monotonic = 1) for i in range(ra.size)]\n",
+    "#make condition on combination and observations\n",
+    "#Do the get-match here in the initialisation. Linear solution for the SEDs.\n",
+    "\n",
+    "rgb = np.zeros((4,n1,n2))\n",
+    "\n",
+    "for src in sources:\n",
```

Comment:
Here we should have a custom `filter_weights` argument for `img_to_rgb` so that HST and HSC filters looks reasonable together. Not critical for this PR